ruby/command-t/scanner/file_scanner.rb

```diff
@@ -81,35 +102,11 @@ def ensure_cache_under_limit
         @paths_keys << @path
       end
 
-      def path_excluded?(path, prefix_len = @prefix_len)
-        if apply_wild_ignore?
```

Comment:
Bah, ignore this comment. Darn Github UI made me think your comment was attached to something else.